modules/planning/toolkits/deciders/side_pass_path_decider.cc

```diff
@@ -128,6 +128,25 @@ bool SidePassPathDecider::DecideSidePassDirection(
   }
 }
 
+bool SidePassPathDecider::GetLaneInfoFromPoint(
+    double point_x, double point_y, double point_z, double point_theta,
```

Comment:
This seems to be a functon in hdmap class.

Reply:
Yes, key to this function is the GetNearestLaneWithHeading function from hdmap. The reason why I wrap it with an extra function here is because: side_pass_path_decider called this multiple times and I wanted to make sure that every time it is called, the other parameters such as the search angle range, etc. are always consistent.